canonical/lxd-ui
# Move storage pool creation form from side panel to dedicated page [WD-6352]

## Done

- Moved storage pool creation form from side panel to dedicated page.

Fixes [WD-6352](https://warthogs.atlassian.net/browse/WD-6352)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Go to storage page
    - Click on the "Create storage pool" button
    - Check the new form in a dedicated page: it should look and behave as expected

[WD-6352]: https://warthogs.atlassian.net/browse/WD-6352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
